Makefile

```diff
@@ -170,16 +199,19 @@ build/.hugo: build/.static build/.pages build/.bibtex build/.mods build/.endnote
 	@echo "INFO     Running Hugo... this may take a while."
 	@cd build && \
 	    hugo -b $(ANTHOLOGYHOST)/$(ANTHOLOGYDIR) \
-	         -d $(ANTHOLOGYDIR) \
+	         -d website/$(ANTHOLOGYDIR) \
 		 -e $(HUGO_ENV) \
 	         --cleanDestinationDir \
 	         --minify
+	@cd build/website/$(ANTHOLOGYDIR) \
+	    && sed -i 's|ANTHOLOGYDIR|$(ANTHOLOGYDIR)|g' .htaccess \
```

Comment:
That backslash at the end seems to be why the build is failing; I guess it's a typo?

Reply:
Yes, the backslash is wrong. There was an additional line that I deleted before commiting because it was a no-op.